docs/nifgen/enums.rst

```diff
@@ -726,6 +726,29 @@ Enums used in NI-FGEN
 
 
 
+.. py:data:: P2PAddressType
+
+    .. py:attribute:: nifgen.P2PAddressType.PHYSICAL
+
+
+
+        Physical
+
+        
+
+
+
+    .. py:attribute:: nifgen.P2PAddressType.VIRTUAL
+
+
+
+        Physical
```

Comment:
I'm guessing this should say Virtual.
Internal bug report?

Reply:
Filed: CAR #685284